src/View/Category/add.html.twig

```diff
@@ -1,4 +1,70 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
+
+<h3 id="titleAddCategory"> Ajouter une catégorie</h3>
```

Comment:
Same thing as Pierre-Louis, id and class should not be written this  way: titleAddCategory 
title-add-category is more conventional.

it goes for the whole file